hardware-and-software-requirements.md

```diff
@@ -18,7 +18,9 @@ TiDB 作为一款开源分布式 NewSQL 数据库，可以很好地部署和运
     |  操作系统   |   支持的 CPU 架构   |
     |   :---   |   :---   |
     | <ul><li>Red Hat Enterprise Linux 8.4 及以上的 8.x 版本</li><li>CentOS 8.4 及以上的 8.x 版本</li></ul>  |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
+    | Red Hat Enterprise Linux 8 Stream |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
```

Comment:
@dveeden PTAL. Adding the Red Hat Enterprise Linux 8 Stream to TiDB-supported Linux OS.

Reply:
Is this RHEL 8 or CentOS 8 Steam? "Red Hat Enterprise Linux 8 Stream" doesn't exist. I assume this is CentOS 8 Stream.

```suggestion
    | CentOS 8 Stream |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
```